utility.py

```diff
@@ -0,0 +1,95 @@
+import os
+import fnmatch
+import zipfile
+import tempfile, shutil
+
+
+def get_input():
+    """
+
+    :return:
+    """
+    file_path = input('Enter the zip file path')
```

Comment:
format input with ": " or similar

Reply:
@wallacerob I have incorporated the changes. Please validated